Statistics/Distributions/LogPearson3.cs

```diff
@@ -134,6 +134,26 @@ private IRange<double> FiniteRange(double min = double.NegativeInfinity, double
             return IRangeFactory.Factory(pmin, pmax);
 
         }
+        /// <summary>
+        /// Added skew validation that steps around a .NET bug where for skewness !=0 and -.002 < skewness < 0.002, an exception is erroneously thrown. 
+        /// </summary>
+        /// <param name="skew"></param>
+        /// <returns></returns>
+        internal double ValidateSkewness(double skew)
+        {
+            if (skew < .002 && skew > 0)
+            {
+                return 0.002;
+            }
+            else if (skew > -.002 && skew < 0)
+            {
+                return -0.002;
```

Comment:
i think you should set skew to zero rather than to an upper or lower end. that would force it to a normal distribution